indico/modules/rb_new/client/js/modules/bookRoom/SearchResultCount.jsx

```diff
@@ -106,6 +108,26 @@ class SearchResultCount extends React.Component {
         );
     }
 
+    renderUnbookable(count) {
+        const label = <Label color="red" horizontal size="small">{count}</Label>;
+        const trigger = (
+            <Menu.Item as="span">
+                <Icon name="lock" />
+                <Translate>
+                    Unauthorized
+                    <Param name="count" value={label} />
```

Comment:
Was it intended to put it in a separate line?

Reply:
There'd be an actual space if it was in the same line (linebreaks between text and elements are ignored in JSX). Not sure if it'd cause any display issues here though. Anyway, nothing wrong with having the param in a separate line IMO.